spec/docs/locations_spec.rb

```diff
@@ -23,6 +23,15 @@
                 type: :string,
                 description: "The code of the course.",
                 example: "X130"
+      parameter name: :include,
```

Comment:
test is failing. i reckon after line 41 if you add `let(:include) { "provider" }` should fix it

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/1523/commits/7493a5ff98c9d75daa6b0af177310715defcfaf6